include/picongpu/plugins/transitionRadiation/Particle.hpp

```diff
@@ -0,0 +1,224 @@
+            float3_X const & locationSet, 
+            float3_X const & momentumSet,
+            float_X const massSet
+        ) :
+            location( locationSet ),
+            momentum( momentumSet ),
+            mass( massSet )
+        { 
+        }
+
+        //! @return momentum
+        HDINLINE 
+        float3_X 
+        getMomentum( ) const
+        {
+            return momentum;
+        }
+
+        //! @return normalized momentum
+        HDINLINE 
+        float_X 
+        getU( ) const
+        {
+            return calcU( getMomentum( ) );
+        }
+
+        //! @return velocity v = beta * c
+        HDINLINE
+        float_X 
+        getVel( ) const
+        {
+            return calcBeta( getMomentum( ) ) * picongpu::SPEED_OF_LIGHT;
+        }
+
+        //! @return polar angle phi of momentum
+        HDINLINE
+        float_X 
+        getMomPhi( ) const
+        {
+            return calcMomPhi( );
+        }
+
+        //! @return azimuth angle psi of momentum
+        HDINLINE
+        float_X 
+        getMomTheta( ) const
+        {
+            return calcMomTheta( );
+        }
+
+        //! @return absolute value of momentum
+        HDINLINE
+        float_X 
+        getMomAbs( ) const
+        {
+            return calcMomAbs( );
+        }
+        
+        //! @return radial, perpendicular component of location in cylindrical coordinates
+        HDINLINE
+        float_X 
+        getPosPerp( ) const
+        {
+            return calcPosRho( );
+        }
+
+        //! @return parallel component to z of location in cylindrical coordinates
+        HDINLINE
+        float_X 
+        getPosPara( ) const
+        {
+            return location.y( );
+        }
+
+        //! @return polar angle of location in cylindrical coordinates
+        HDINLINE
+        float_X 
+        getPosPhi( ) const
+        {
+            return calcPosPhi( );
+        }
+
+    private:
+        //! @return beta=v/c
+        HDINLINE 
+        float_X 
+        calcBeta(
+            float3_X const & momentum
+        ) const
+        {
+            float_X const gamma = calcGamma(momentum);
+            return picongpu::math::sqrt(1 - 1 / (gamma * gamma) );
+        }
+
+        //! @return gamma = E/(mc^2)
+        HDINLINE 
+        float_X 
+        calcGamma(
+            float3_X const & momentum
+        ) const
+        {
+            float_X const x = 1.0 / ( mass * picongpu::SPEED_OF_LIGHT ) * ( mass * picongpu::SPEED_OF_LIGHT );
+            return picongpu::math::sqrt( 
+                1.0 + ( momentum * momentum ).sumOfComponents( ) * x 
+            );
+        }
+
+        //! @return normalized momentum u = gamma * beta
+        HDINLINE 
+        float_X 
+        calcU(
+            float3_X const & momentum
+        ) const
+        {
+            float_X const gamma1 = calcGamma( momentum );
+            float_X const beta1 = calcBeta( momentum );
+            return gamma1 * beta1;
```

Comment:
why do you use the additional `...1` in the variable names?
If there is a deeper meaning, please add a comment - if not, please remove the suffix to avoid confusion.